tests/test_dfp_kafka.py

```diff
@@ -106,14 +105,6 @@ def test_dfp_roleg(mock_ae: mock.MagicMock,
     pipe.add_stage(preprocess_ae_stage.PreprocessAEStage(config))
     pipe.add_stage(AutoEncoderInferenceStage(config))
     pipe.add_stage(AddScoresStage(config))
-    pipe.add_stage(
```

Comment:
How are we able to just remove this? Was it unnecessary?

Reply:
@cwharris I removed it because i changed the accepted type for `TimeSeriesStage` stage to `MultiAEMessage` which no longer made it usable in this Starter DFP pipeline. Realized that I can use `MultiMessage` instead which both `MuitiAEMessage` and `MultiResponseMessage` inherit from. So yes, you're right. It is unnecessary. I'll revert it after making the changes. Thanks!